marimo/_plugins/ui/_impl/table.py

```diff
@@ -61,6 +61,12 @@ class ColumnSummaries:
     summaries: List[ColumnSummary]
 
 
+@dataclass
+class SortValuesArgs:
+    by: Optional[str]
```

Comment:
what does optional mean? should this be ` by: str`?

Reply:
For a list of values (i.e. 1D list with no column name), I meant to set `by` to `null` so that the `sort_values` function would just sort the list.

Another approach could be to pass a custom `by` value, but there's the one in million chance that someone could use that as a column name.

A third arg seemed like overkill

Reply:
when we pass a list of single values, we actually add a column `"value"`. that can be the sortable column. 

so you could do something like `return sorted(self._normalize_data(self.data), ...)` and you will always have a column

Reply:
Worked like a charm! Thank you